test/C/src/multiport/MultiportFromBank.lf

```diff
@@ -2,7 +2,7 @@
  // Here, the bank is smaller than the width of the sending port.
 target C {
     timeout: 2 sec,
-    threads: 4,
+    workers: 4,
```

Comment:
Generally speaking, it seems odd to select a specific number of threads for tests. Should we just slash all of these and leave it up to the runtime?